cmd/utils.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/redhat-developer/ocdev/pkg/component"
+	"github.com/redhat-developer/ocdev/pkg/occlient"
+	"github.com/redhat-developer/ocdev/pkg/storage"
+	"github.com/redhat-developer/ocdev/pkg/url"
+)
+
+// printDeleteAppInfo will print things which will be deleted
+func printDeleteAppInfo(client *occlient.Client, appName string) error {
+	componentList, err := component.List(client)
+	if err != nil {
+		return errors.Wrap(err, "Failed to get Component list")
+	}
+
+	for _, cmpnt := range componentList {
+		fmt.Println("Component", cmpnt.Name, "will be deleted:")
+		appUrl, err := url.List(client, cmpnt.Name, appName)
+		if err != nil {
+			return errors.Wrap(err, "Failed to get URL list")
+		}
+		if len(appUrl) != 0 {
+			fmt.Println("This component is externally exposed, and the URL will be removed")
+		}
+		appStore, err := storage.List(client, &occlient.VolumeConfig{
```

Comment:
#295 changes `storage.List`, should we wait for that PR? 
Or changing this will be handled in #295 

#295 is already big PR, so it might make sense to wait for it.
What do you think @surajnarwade @containscafeine ?